packages/demo/resources/plasma-logo.svg

```diff
@@ -0,0 +1,90 @@
+<svg width="229" height="102" viewBox="0 0 229 102" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
I put the svg through https://jakearchibald.github.io/svgomg/ and was able to reduce the size quite a bit without noticing any change on the svg.

```svg
<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 229 102">
  <path fill="url(#a)" d="M54.9377 51.0834c.0002-16.5161-9.141-27.4251-13.8645-31.9596-1.0974-1.0534-2.7548-.2527-2.7548 1.2685v61.2292c0 1.515 1.6464 2.3173 2.7457 1.2749 4.7207-4.4763 13.8734-15.2862 13.8736-31.813Z"/>
  <path fill="url(#b)" d="M44.341 51.0725c.0002-13.8782-7.4626-23.1787-11.6335-27.2929-1.083-1.0683-2.7399-.2783-2.7399 1.243v51.9719c0 1.515 1.6454 2.3073 2.7303 1.25 4.169-4.0628 11.643-13.2833 11.6431-27.172Z"/>
  <path fill="url(#c)" d="M34.2677 51.0554c.0001-10.0681-5.0981-17.0089-8.3759-20.431-1.0523-1.0986-2.6973-.3198-2.6973 1.2014V70.194c0 1.515 1.6342 2.2956 2.6885 1.2077 3.2774-3.3822 8.3846-10.2693 8.3847-20.3463Z"/>
  <path fill="url(#d)" d="M26.1723 51.0388c.0001-7.1969-3.38-12.3206-5.8858-15.1301-1.0125-1.1353-2.6485-.374-2.6485 1.1472l-.0001 27.967c0 1.5213 1.6363 2.2826 2.6488 1.147 2.5057-2.8101 5.8855-7.9344 5.8856-15.1311Z"/>
  <g filter="url(#e)">
    <path fill="url(#f)" d="M54.9372 51.0834c.0002-16.5161-9.141-27.4251-13.8645-31.9596-1.0974-1.0534-2.7548-.2527-2.7548 1.2685v61.2292c0 1.515 1.6464 2.3173 2.7457 1.2749 4.7207-4.4763 13.8735-15.2862 13.8736-31.813Z"/>
  </g>
  <g filter="url(#g)" opacity=".7" style="mix-blend-mode:overlay">
    <path fill="url(#h)" d="M45.8586 51.0725c.0002-19.2604-14.3735-29.704-14.3735-29.704v59.2471s14.3733-10.2828 14.3735-29.5431Z"/>
  </g>
  <g filter="url(#i)" opacity=".7" style="mix-blend-mode:hard-light">
    <path fill="url(#j)" d="M35.1984 51.0554c.0001-14.838-11.0732-22.8837-11.0732-22.8837v45.6435s11.073-7.9218 11.0732-22.7598Z"/>
  </g>
  <g filter="url(#k)" opacity=".7" style="mix-blend-mode:screen">
    <path fill="url(#l)" d="M32.6884 51.0388c.0001-7.1969-3.3799-12.3206-5.8857-15.1301-1.0126-1.1353-2.6486-.374-2.6486 1.1472l-.0001 27.967c0 1.5213 1.6363 2.2826 2.6488 1.147 2.5057-2.8101 5.8855-7.9344 5.8856-15.1311Z"/>
  </g>
  <path fill="#fff" d="M84.8371 45.555c0 1.0892-.2057 2.0331-.6172 2.8318-.3872.7745-.9439 1.4764-1.67 2.1057-.7261.6293-1.5732 1.077-2.5414 1.3433-.9681.2662-2.0451.3993-3.2311.3993h-3.7757v11.6902h-1.9605V38.8749h5.7362c1.186 0 2.263.1331 3.2311.3994.9682.2662 1.8153.714 2.5414 1.3433.7261.6292 1.2828 1.3432 1.67 2.1419.4115.7746.6172 1.7064.6172 2.7955Zm-2.0694 0c0-.8955-.1573-1.6458-.472-2.2509-.3146-.6293-.7503-1.1375-1.3069-1.5248-.5325-.3872-1.1618-.6656-1.8879-.835-.7019-.1936-1.4522-.2904-2.2509-.2904h-3.8483v9.8023H76.85c.7987 0 1.549-.0847 2.2509-.2542.7261-.1936 1.3554-.484 1.8879-.8713.5566-.3872.9923-.8834 1.3069-1.4885.3147-.6293.472-1.3917.472-2.2872Zm6.426 18.3703V38.8749h1.9604v23.2715h8.9309v1.7789H89.1937Zm31.9953 0-3.086-7.5151h-11.944l-3.122 7.5151h-2.07l10.71-25.123h1.089l10.71 25.123h-2.287Zm-9.076-22.146-5.228 12.8156h10.456l-5.228-12.8156Zm28.149 15.6111c0 .9439-.194 1.8395-.581 2.6866-.363.8471-.896 1.5974-1.597 2.2509-.702.6293-1.549 1.1254-2.542 1.4885-.992.363-2.093.5446-3.304.5446-1.21 0-2.275-.1816-3.194-.5446-.896-.3873-1.658-.8229-2.288-1.307-.605-.5083-1.089-1.0165-1.452-1.5248-.339-.5083-.581-.8834-.726-1.1255l1.525-1.0891c.218.4115.52.8471.908 1.307.387.4598.847.8834 1.379 1.2706.533.3631 1.138.6777 1.815.944.678.242 1.416.363 2.215.363.799 0 1.549-.121 2.251-.363.726-.2421 1.355-.593 1.888-1.0529.532-.4598.944-1.0044 1.234-1.6337.315-.6535.472-1.3917.472-2.2146s-.181-1.5127-.544-2.0694c-.364-.5808-.799-1.0649-1.307-1.4522-.509-.4114-1.041-.7382-1.598-.9802-.532-.242-.98-.4356-1.343-.5809-.726-.2904-1.464-.5929-2.215-.9076-.75-.3388-1.379-.6414-1.888-.9076-.242-.1452-.568-.351-.98-.6172-.411-.2904-.811-.6656-1.198-1.1254-.387-.4599-.726-1.0287-1.016-1.7064-.291-.6777-.436-1.4643-.436-2.3598 0-.9197.181-1.7547.544-2.505.388-.7503.908-1.3917 1.562-1.9242.653-.5567 1.428-.9802 2.323-1.2707.92-.3146 1.9-.4719 2.941-.4719 1.065 0 1.984.1573 2.759.4719.799.2905 1.464.6293 1.997 1.0166.556.3872.98.7624 1.27 1.1254.291.3389.46.5567.509.6535l-1.38 1.1981c-.242-.2905-.52-.593-.835-.9076-.29-.3389-.653-.6414-1.089-.9077-.412-.2662-.896-.484-1.452-.6535-.557-.1694-1.198-.2541-1.924-.2541-.727 0-1.404.0968-2.033.2904-.63.1937-1.186.472-1.671.8351-.459.363-.835.8229-1.125 1.3795-.266.5325-.399 1.1497-.399 1.8516s.109 1.307.326 1.8152c.218.5083.509.944.872 1.307.363.3631.762.6777 1.198.9439.46.2421.907.4599 1.343.6535.436.1937.92.4115 1.452.6535.557.2178 1.077.4357 1.561.6535.509.1936.956.3873 1.344.5809.387.1694.653.2904.798.363.146.0726.412.2421.799.5083.412.2662.823.6293 1.234 1.0891.412.4599.775 1.0408 1.09 1.7427.338.7019.508 1.5248.508 2.4687Zm28.371 6.5349-2.142-21.7466-9.258 22.1823h-.182l-9.294-22.2186-2.069 21.7829h-1.96l2.541-25.0504h2.251l8.604 21.3836 8.641-21.3836h2.287l2.541 25.0504h-1.96Zm24.173 0-3.086-7.5151h-11.944l-3.122 7.5151h-2.07l10.71-25.123h1.089l10.71 25.123h-2.287Zm-9.076-22.146-5.228 12.8156h10.456l-5.228-12.8156Z"/>
  <g clip-path="url(#m)">
    <path fill="#fff" d="m173.21 74.7579-.445.4723c-.309-.2997-.636-.4723-1.081-.4723-.772 0-1.353.6085-1.353 1.4622 0 .8537.581 1.4622 1.353 1.4622.418 0 .818-.1725 1.108-.4722l.418.4722c-.363.4178-.881.6358-1.49.6358-1.271 0-2.097-.9082-2.097-2.098 0-1.2169.826-2.1251 2.097-2.1251.573-.0091 1.118.2361 1.49.6629ZM178.178 76.2199c0 1.1897-.908 2.0979-2.125 2.0979-1.245 0-2.153-.9082-2.153-2.0979 0-1.217.908-2.1252 2.153-2.1252 1.217 0 2.125.9082 2.125 2.1252Zm-3.56 0c0 .8537.608 1.4622 1.435 1.4622.799 0 1.407-.6085 1.407-1.4622 0-.8537-.608-1.4622-1.407-1.4622-.827 0-1.435.6085-1.435 1.4622ZM181.165 78.236h-.835l-1.707-4.0324h.772l1.353 3.3422 1.353-3.3422h.745l-1.681 4.0324ZM187.205 76.1924v.2725h-3.179c.082.8264.663 1.2442 1.354 1.2442.463.0091.917-.1635 1.271-.4723l.363.4723c-.499.4723-1.053.6176-1.662.6176-1.217 0-2.07-.8265-2.07-2.098 0-1.2442.853-2.1251 2.016-2.1251 1.108-.0091 1.907.8718 1.907 2.0888Zm-3.169-.3088h2.488c-.082-.6902-.554-1.1624-1.217-1.1624-.717.009-1.162.4722-1.271 1.1624ZM192.4 76.2199c0 1.1897-.908 2.0979-2.125 2.0979-1.244 0-2.153-.9082-2.153-2.0979 0-1.217.909-2.1252 2.153-2.1252 1.208 0 2.125.9082 2.125 2.1252Zm-3.56 0c0 .8537.608 1.4622 1.435 1.4622.799 0 1.407-.6085 1.407-1.4622 0-.8537-.608-1.4622-1.407-1.4622-.836 0-1.435.6085-1.435 1.4622Z"/>
    <path fill="#00ADFF" d="M166.253 72.8234c-.781-.7719-1.798-1.2442-2.897-1.3532-.081 0-.145.0727-.145.1635 0 .0363.009.0636.036.0908l1.708 1.6802c0 .0272 0 .0272-.028.0272-.472-.3451-1.026-.554-1.598-.6085-.054-.0091-.1.0273-.109.0818 0 .0272.009.0544.027.0817l1.163 1.1625s0 .0272-.027 0c-.318-.1998-.673-.327-1.054-.3633-.054 0-.082.0545-.054.109l1.961 1.9617c.055.0545.136 0 .109-.0545-.027-.3724-.154-.7357-.363-1.0535-.027 0 0-.0273 0 0l1.135 1.108c.055.0545.164 0 .164-.0818-.064-.5812-.273-1.1443-.609-1.6256-.027 0 0-.0273 0 0l1.717 1.6801c.045.0545.118.0636.172.0273.046-.0273.064-.0818.046-.1363-.1-1.0989-.582-2.1251-1.354-2.8971Z"/>
    <path fill="#F05245" d="M159.605 72.8234c.79-.7629 1.807-1.2442 2.897-1.3532.082 0 .146.0545.155.1362 0 .0364-.009.0818-.046.109l-1.68 1.6893c-.027.0272 0 .0272 0 .0272.481-.3451 1.045-.554 1.626-.6085.082 0 .136.109.082.1635l-1.154 1.1625c-.027 0 0 .0272 0 0 .318-.1998.672-.3179 1.054-.3633.054 0 .109.0545.054.109l-1.971 1.9617c-.054.0545-.109 0-.109-.0545.037-.3724.173-.7357.391-1.0535 0 0-.027-.0273-.027 0l-1.108 1.108c-.082.0545-.191 0-.164-.0818.055-.5903.264-1.1534.609-1.6256 0 0-.028-.0273-.028 0l-1.689 1.6801c-.109.109-.245.0273-.245-.109.109-1.0898.59-2.107 1.353-2.8971Z"/>
    <path fill="#1CEBCF" d="M159.605 79.5348c.772.781 1.798 1.2623 2.897 1.3532.073.009.137-.0364.146-.109.009-.0454-.009-.0818-.037-.1181l-1.68-1.7074c-.027 0 0-.0272 0 0 .481.3451 1.045.554 1.626.5994.082 0 .136-.109.082-.1635l-1.154-1.1534c-.027-.0272 0-.0272 0-.0272.309.2089.672.3451 1.045.3814.054 0 .109-.0817.054-.1089l-1.962-1.9617c-.054-.0545-.109-.0273-.109.0545.037.3723.173.7356.391 1.0535 0 0-.027.0272-.027 0l-1.108-1.108c-.082-.0545-.191-.0273-.164.0817.055.5812.264 1.1352.609 1.5984 0 .0182-.009.0273-.028.0273l-1.689-1.7074c-.063-.0545-.163-.0364-.209.0272-.018.0273-.027.0545-.036.0908.1 1.0899.581 2.1161 1.353 2.8972Z"/>
    <path fill="#FFE300" d="M166.253 79.5347c-.772.7901-1.798 1.2715-2.897 1.3532-.109.0273-.191-.1362-.109-.2179l1.707-1.7074s0-.0273-.027 0c-.472.3451-1.026.554-1.598.6085-.055.009-.1-.0364-.109-.0909 0-.0272.009-.0545.027-.0817l1.163-1.1625c0-.0272 0-.0272-.028-.0272-.308.2179-.672.3542-1.044.3814-.046-.0091-.073-.0454-.064-.0817 0-.0091 0-.0182.009-.0273l1.962-1.9617c.055-.0545.136-.0272.109.0545-.027.3724-.154.7357-.363 1.0535-.027 0 0 .0273 0 0l1.135-1.108c.036-.0363.1-.0363.136 0 .018.0182.028.0545.028.0818-.064.5721-.273 1.1261-.609 1.5984-.027.0272 0 .0272 0 .0272l1.707-1.7074c.082-.0817.246 0 .218.109-.09 1.0989-.572 2.1252-1.353 2.9062Z"/>
    <path fill="#fff" d="M193.744 74.3671v1.0081h-.172v-1.0081h-.318v-.1635h.817v.1635h-.327ZM195.279 75.3754v-.7811l-.272.5631h-.128l-.272-.5631v.7811h-.182v-1.1625h.182l.336.7265.336-.7265h.173v1.1625h-.173Z" opacity=".87"/>
  </g>
  <path fill="#6B6BBD" d="M101.262 73.7582h1.668c.435 0 .837.067 1.206.2011.374.134.695.3267.964.5781.268.2513.477.5557.628.9132.156.3574.234.7596.234 1.2064 0 .4357-.078.8322-.234 1.1897-.157.3519-.372.6535-.645.9048-.274.2514-.595.4469-.964.5865-.368.134-.765.2011-1.189.2011h-1.668v-5.7809Zm1.651 5.2279c.346 0 .664-.0558.955-.1675.296-.1118.55-.2681.762-.4692.213-.2067.377-.4524.495-.7373.123-.2904.184-.6116.184-.9635 0-.363-.059-.6898-.176-.9802-.112-.2904-.274-.5362-.486-.7373-.207-.201-.458-.3546-.754-.4608-.296-.1061-.623-.1591-.98-.1591h-1.022v4.6749h1.022Zm7.328-.0084c-.363.4301-.93.6451-1.701.6451-.29 0-.556-.0474-.796-.1424-.234-.0949-.438-.2318-.611-.4105-.168-.1787-.299-.3938-.394-.6451-.09-.2569-.134-.5446-.134-.863 0-.2792.044-.5417.134-.7875.095-.2457.226-.4608.394-.6451.167-.1843.371-.3295.611-.4357.24-.1061.506-.1591.796-.1591.268 0 .514.0474.737.1424.229.0893.422.2178.578.3854.162.1619.288.3574.377.5864.09.229.134.4776.134.7457 0 .0335-.002.0698-.008.1089v.1089c-.005.0391-.011.0754-.017.1089h-3.1c0 .2011.034.3882.101.5614.073.1675.17.3127.293.4356.123.1229.271.2178.444.2849.173.067.363.1005.57.1005.195 0 .371-.0251.528-.0754.162-.0503.304-.1089.427-.1759.134-.0671.257-.148.369-.243l.268.3686Zm-.503-1.7258c0-.1397-.025-.2793-.075-.4189-.045-.1397-.118-.2654-.218-.3771-.095-.1173-.221-.2122-.377-.2848-.151-.0782-.335-.1173-.553-.1173-.196 0-.371.0335-.528.1005-.151.0671-.279.1564-.385.2681-.101.1117-.182.2402-.243.3854-.062.1452-.098.2933-.109.4441h2.488Zm1.437 1.2818c.19.2122.38.3658.57.4608.196.0894.388.134.578.134.106 0 .21-.0111.31-.0335.101-.0279.19-.067.268-.1173.079-.0502.14-.1173.185-.201.05-.0838.075-.1844.075-.3016 0-.1173-.036-.2151-.109-.2933-.072-.0838-.159-.1536-.26-.2094-.1-.0559-.206-.1006-.318-.1341-.106-.0391-.198-.0698-.276-.0921-.073-.0224-.174-.0531-.302-.0922-.123-.0447-.246-.1061-.369-.1843-.123-.0838-.229-.1927-.318-.3268-.089-.134-.134-.3044-.134-.511 0-.1788.036-.3351.109-.4692.072-.1396.167-.2541.285-.3435.117-.0949.251-.1648.402-.2094.156-.0503.313-.0754.469-.0754.318 0 .578.0446.779.134.201.0894.372.1983.511.3267l-.301.3771c-.14-.1173-.296-.2067-.47-.2681-.167-.0615-.34-.0922-.519-.0922-.207 0-.374.0475-.503.1424-.128.095-.192.2346-.192.4189 0 .0894.022.1676.067.2346.05.067.111.1285.184.1843.078.0503.168.0978.268.1425.106.0391.212.0754.318.1089.101.0335.221.0754.361.1256.145.0447.285.1118.419.2011.134.0838.248.1955.343.3351.095.1341.142.3072.142.5195 0 .1899-.036.3602-.108.511-.073.1453-.176.2709-.31.3771-.129.1005-.282.1759-.461.2262-.173.0558-.36.0837-.562.0837-.323 0-.611-.0614-.862-.1843-.252-.1284-.45-.2988-.595-.511l.326-.3938Zm4.209-4.2728c-.005.1229-.047.2262-.126.31-.078.0782-.181.1173-.31.1173-.128 0-.234-.0391-.318-.1173-.084-.0838-.126-.1871-.126-.31 0-.1229.042-.2234.126-.3016.084-.0838.19-.1257.318-.1257.129 0 .232.0419.31.1257.084.0782.126.1787.126.3016Zm-.746 1.3489h.62v3.9293h-.62v-3.9293Zm4.882 3.1585c-.145.2346-.338.4189-.578.5529-.235.1285-.52.1927-.855.1927-.257 0-.503-.0419-.737-.1256-.229-.0838-.43-.2095-.603-.3771-.168-.1675-.302-.3742-.403-.6199-.1-.2514-.15-.539-.15-.863 0-.3072.05-.5836.15-.8294.101-.2458.235-.4552.403-.6284.173-.1731.371-.3044.594-.3937.229-.095.472-.1424.729-.1424.363 0 .665.0754.905.2262.246.1452.43.3211.553.5278h.017l-.008-.6786h.62v3.0915c0 .4412-.031.7903-.093 1.0472-.061.2569-.156.472-.285.6451-.061.0894-.139.1815-.234.2765-.095.095-.212.1815-.352.2597-.134.0782-.296.1424-.486.1927-.184.0503-.402.0754-.653.0754-.246 0-.472-.0307-.679-.0921-.201-.0559-.374-.1201-.519-.1927-.146-.0671-.263-.1341-.352-.2011-.084-.0614-.134-.0978-.151-.1089l.31-.4441c.067.0503.148.1034.243.1592.1.0615.212.1173.335.1676.123.0558.257.1005.402.134.145.0335.293.0503.444.0503.268 0 .494-.0419.679-.1257.19-.0838.34-.201.452-.3518.112-.1453.193-.3212.243-.5279.05-.2066.076-.4328.076-.6786v-.2178h-.017Zm-1.324.2262c.223 0 .419-.0363.587-.1089.173-.0782.315-.1843.427-.3184.117-.134.204-.2904.26-.4692.061-.1787.092-.3714.092-.578 0-.4525-.126-.8071-.377-1.0641-.246-.2625-.581-.3937-1.006-.3937-.223 0-.419.0391-.586.1173-.168.0726-.307.1759-.419.31-.112.1284-.198.282-.26.4607-.056.1788-.084.3687-.084.5698 0 .2066.028.3993.084.578.062.1788.148.3352.26.4692.117.1341.26.2402.427.3184.173.0726.372.1089.595.1089Zm5.011-3.4601c.279 0 .508.0418.687.1256.184.0782.326.1899.427.3351.106.1453.179.3184.218.5195.039.1955.058.4105.058.6451v2.3794h-.62v-2.3375c0-.3686-.069-.6479-.209-.8378-.134-.1955-.385-.2933-.754-.2933-.173 0-.327.0336-.461.1006s-.246.1592-.335.2765c-.084.1117-.148.2457-.193.4021-.044.1508-.067.31-.067.4776v2.2118h-.62v-3.9293h.62v.6367h.025c.09-.2123.243-.3826.461-.5111.224-.134.478-.201.763-.201Zm4.42 2.6642c.061.0949.142.1955.243.3016.1.1061.215.2039.343.2932.129.0838.265.1564.411.2178.145.0559.296.0838.452.0838.352 0 .64-.0865.863-.2597.223-.1787.335-.4329.335-.7624 0-.1508-.028-.2821-.084-.3938-.055-.1173-.131-.2206-.226-.31-.095-.0893-.204-.1675-.327-.2345-.122-.0726-.251-.1369-.385-.1927l-.855-.3687c-.323-.1396-.566-.3211-.728-.5445-.162-.2235-.243-.5111-.243-.863 0-.2457.05-.4608.15-.6451.101-.1843.235-.3379.403-.4608.167-.1229.357-.215.569-.2765.213-.067.43-.1005.654-.1005.24 0 .452.0335.637.1005.189.0671.349.1425.477.2263.134.0837.24.1675.318.2513.079.0782.129.134.151.1676l-.368.4189c-.123-.1453-.285-.2849-.486-.4189-.201-.1341-.456-.2011-.763-.2011-.313 0-.575.0782-.787.2346-.213.1564-.319.377-.319.6618 0 .1508.023.2793.067.3854.051.1061.112.1983.185.2765.078.0726.164.1368.259.1927.101.0503.199.0949.294.134l.787.3352c.179.0726.344.1564.495.2513.156.0894.29.1955.402.3184.111.1228.201.2625.268.4189.067.1564.1.3379.1.5445 0 .257-.047.4888-.142.6954-.095.2011-.229.3742-.402.5195-.168.1396-.369.2485-.604.3267-.234.0782-.491.1173-.77.1173-.218 0-.422-.0335-.612-.1005-.19-.0671-.36-.1508-.511-.2514-.151-.1005-.285-.2094-.402-.3267-.112-.1229-.204-.2402-.277-.3519l.428-.4105Zm3.573-2.5972.646.0084 1.273 3.2339 1.248-3.2423.645.0084-1.918 4.6414c-.073.1787-.148.3267-.226.444-.073.1173-.148.2095-.227.2765-.078.0726-.162.1229-.251.1508-.084.0279-.179.0419-.285.0419-.089 0-.17-.0084-.243-.0251-.072-.0112-.137-.0252-.192-.0419-.067-.0224-.126-.0447-.176-.067l.125-.5279c.045.0168.09.0336.134.0503.039.0112.081.0196.126.0251.045.0112.089.0168.134.0168.05 0 .098-.0028.143-.0084.044-.0056.089-.0279.134-.067.044-.0335.089-.0866.134-.1592.05-.0726.103-.1759.159-.31l.218-.511-1.601-3.9377Zm4.218 2.9323c.19.2122.38.3658.569.4608.196.0894.389.134.579.134.106 0 .209-.0111.31-.0335.1-.0279.189-.067.268-.1173.078-.0502.139-.1173.184-.201.05-.0838.075-.1844.075-.3016 0-.1173-.036-.2151-.109-.2933-.072-.0838-.159-.1536-.259-.2094-.101-.0559-.207-.1006-.319-.1341-.106-.0391-.198-.0698-.276-.0921-.073-.0224-.173-.0531-.302-.0922-.123-.0447-.245-.1061-.368-.1843-.123-.0838-.229-.1927-.319-.3268-.089-.134-.134-.3044-.134-.511 0-.1788.037-.3351.109-.4692.073-.1396.168-.2541.285-.3435.117-.0949.251-.1648.402-.2094.157-.0503.313-.0754.469-.0754.319 0 .578.0446.779.134.202.0894.372.1983.512.3267l-.302.3771c-.14-.1173-.296-.2067-.469-.2681-.168-.0615-.341-.0922-.52-.0922-.206 0-.374.0475-.502.1424-.129.095-.193.2346-.193.4189 0 .0894.022.1676.067.2346.05.067.112.1285.184.1843.078.0503.168.0978.268.1425.107.0391.213.0754.319.1089.1.0335.22.0754.36.1256.145.0447.285.1118.419.2011.134.0838.248.1955.343.3351.095.1341.143.3072.143.5195 0 .1899-.036.3602-.109.511-.073.1453-.176.2709-.31.3771-.129.1005-.282.1759-.461.2262-.173.0558-.36.0837-.561.0837-.324 0-.612-.0614-.863-.1843-.251-.1284-.45-.2988-.595-.511l.327-.3938Zm5.18.9635c-.184.0838-.41.1256-.678.1256-.235 0-.416-.0474-.545-.1424-.123-.1005-.215-.229-.276-.3854-.056-.1564-.09-.3267-.101-.511-.011-.1899-.017-.3743-.017-.553v-1.8934h-.586v-.5195h.586v-1.2148h.629v1.2148h.963v.5195h-.972v2.0191c0 .162.006.31.017.444.011.1285.048.2402.109.3351.033.0503.084.0894.151.1173.072.0224.156.0335.251.0335.062 0 .117-.0027.168-.0083.056-.0112.103-.0252.142-.0419.045-.0112.087-.0252.126-.0419l.033.5027Zm4.071-.5195c-.363.4301-.93.6451-1.7.6451-.291 0-.556-.0474-.796-.1424-.235-.0949-.439-.2318-.612-.4105-.167-.1787-.299-.3938-.394-.6451-.089-.2569-.134-.5446-.134-.863 0-.2792.045-.5417.134-.7875.095-.2457.227-.4608.394-.6451.168-.1843.372-.3295.612-.4357.24-.1061.505-.1591.796-.1591.268 0 .513.0474.737.1424.229.0893.422.2178.578.3854.162.1619.288.3574.377.5864.089.229.134.4776.134.7457 0 .0335-.003.0698-.008.1089v.1089c-.006.0391-.012.0754-.017.1089h-3.1c0 .2011.034.3882.101.5614.072.1675.17.3127.293.4356.123.1229.271.2178.444.2849.173.067.363.1005.57.1005.195 0 .371-.0251.527-.0754.162-.0503.305-.1089.428-.1759.134-.0671.257-.148.368-.243l.268.3686Zm-.502-1.7258c0-.1397-.025-.2793-.076-.4189-.044-.1397-.117-.2654-.218-.3771-.095-.1173-.22-.2122-.377-.2848-.15-.0782-.335-.1173-.553-.1173-.195 0-.371.0335-.527.1005-.151.0671-.28.1564-.386.2681-.1.1117-.181.2402-.243.3854-.061.1452-.097.2933-.109.4441h2.489Zm4.47-.8714c.084-.2457.232-.4468.444-.6032.218-.162.466-.2429.746-.2429.284 0 .516.0474.695.1424.179.0893.318.2122.419.3686.1.1508.167.324.201.5195.033.1954.05.3965.05.6032v2.371h-.62v-2.3375c0-.1173-.011-.243-.033-.377-.017-.1341-.056-.257-.118-.3687-.061-.1117-.145-.2038-.251-.2764-.106-.0726-.249-.109-.427-.109-.162 0-.307.0364-.436.109-.123.0726-.226.1731-.31.3016-.084.1284-.148.2765-.193.444-.044.1676-.067.3491-.067.5446v2.0694h-.62v-2.3375c0-.1173-.011-.243-.033-.377-.017-.1341-.056-.257-.117-.3687-.062-.1117-.146-.2038-.252-.2764-.106-.0726-.248-.109-.427-.109-.162 0-.307.0364-.436.109-.123.0726-.229.1731-.318.3016-.084.1284-.148.2765-.193.444-.044.1676-.067.3491-.067.5446v2.0694h-.62v-3.9293h.62l-.008.6283h.025c.095-.2011.237-.3686.427-.5027.196-.134.414-.201.654-.201.385 0 .673.0809.863.2429.195.1564.329.3575.402.6032Z"/>
  <defs>
    <linearGradient id="a" x1="53.1097" x2="59.112" y1="23.9945" y2="80.3136" gradientUnits="userSpaceOnUse">
      <stop stop-color="#F05245"/>
      <stop offset="1" stop-color="#F05245" stop-opacity=".08"/>
    </linearGradient>
    <linearGradient id="b" x1="42.76" x2="55.8537" y1="27.6442" y2="71.4703" gradientUnits="userSpaceOnUse">
      <stop stop-color="#FFE300"/>
      <stop offset="1" stop-color="#FFE300" stop-opacity="0"/>
    </linearGradient>
    <linearGradient id="c" x1="33.0497" x2="35.374" y1="33.0065" y2="71.4699" gradientUnits="userSpaceOnUse">
      <stop stop-color="#16DCC1"/>
      <stop offset="1" stop-color="#16DCC1" stop-opacity="0"/>
    </linearGradient>
    <linearGradient id="d" x1="25.2335" x2="24.2032" y1="37.128" y2="66.8159" gradientUnits="userSpaceOnUse">
      <stop stop-color="#00ADFF"/>
      <stop offset="1" stop-color="#00ADFF" stop-opacity="0"/>
    </linearGradient>
    <linearGradient id="f" x1="53.1092" x2="48.4548" y1="23.9945" y2="93.3461" gradientUnits="userSpaceOnUse">
      <stop stop-color="#F05245"/>
      <stop offset="1" stop-color="#F05245" stop-opacity="0"/>
    </linearGradient>
    <linearGradient id="h" x1="42.0116" x2="40.1498" y1="25.391" y2="91.4845" gradientUnits="userSpaceOnUse">
      <stop stop-color="#FFE300"/>
      <stop offset="1" stop-color="#FFE300" stop-opacity="0"/>
    </linearGradient>
    <linearGradient id="j" x1="27.4611" x2="25.5993" y1="32.8378" y2="80.7788" gradientUnits="userSpaceOnUse">
      <stop stop-color="#1CEBCF"/>
      <stop offset="1" stop-color="#1CEBCF" stop-opacity="0"/>
    </linearGradient>
    <linearGradient id="l" x1="24.2031" x2="30.7193" y1="36.0964" y2="68.6777" gradientUnits="userSpaceOnUse">
      <stop stop-color="#00ADFF"/>
      <stop offset="1" stop-color="#00ADFF" stop-opacity="0"/>
    </linearGradient>
    <filter id="e" width="53.8552" height="101.958" x="19.7" y=".0271339" color-interpolation-filters="sRGB" filterUnits="userSpaceOnUse">
      <feFlood flood-opacity="0" result="BackgroundImageFix"/>
      <feBlend in="SourceGraphic" in2="BackgroundImageFix" result="shape"/>
      <feGaussianBlur result="effect1_foregroundBlur_1045_19274" stdDeviation="9.30894"/>
    </filter>
    <filter id="g" width="19.9589" height="64.8324" x="28.6924" y="18.576" color-interpolation-filters="sRGB" filterUnits="userSpaceOnUse">
      <feFlood flood-opacity="0" result="BackgroundImageFix"/>
      <feBlend in="SourceGraphic" in2="BackgroundImageFix" result="shape"/>
      <feGaussianBlur result="effect1_foregroundBlur_1045_19274" stdDeviation="1.39634"/>
    </filter>
    <filter id="i" width="25.9676" height="60.5374" x="16.6781" y="20.7247" color-interpolation-filters="sRGB" filterUnits="userSpaceOnUse">
      <feFlood flood-opacity="0" result="BackgroundImageFix"/>
      <feBlend in="SourceGraphic" in2="BackgroundImageFix" result="shape"/>
      <feGaussianBlur result="effect1_foregroundBlur_1045_19274" stdDeviation="3.72358"/>
    </filter>
    <filter id="k" width="40.1848" height="63.0025" x="8.32885" y="19.5381" color-interpolation-filters="sRGB" filterUnits="userSpaceOnUse">
      <feFlood flood-opacity="0" result="BackgroundImageFix"/>
      <feBlend in="SourceGraphic" in2="BackgroundImageFix" result="shape"/>
      <feGaussianBlur result="effect1_foregroundBlur_1045_19274" stdDeviation="7.9126"/>
    </filter>
    <clipPath id="m">
      <path fill="#fff" d="M0 0h37.2358v9.44517H0z" transform="translate(158.252 71.4702)"/>
    </clipPath>
  </defs>
</svg>
```

Reply:
I don't know what that means :sweat_smile: